files/controller-installation/deploy-v1.0.6.yaml

```diff
@@ -7861,9 +8828,55 @@ spec:
           allowPrivilegeEscalation: false
           capabilities:
             drop:
-              - ALL
+            - ALL
           readOnlyRootFilesystem: true
+        volumeMounts:
+        - mountPath: /etc/webhook-cert
+          name: webhook-cert
+          readOnly: true
       securityContext:
         runAsNonRoot: true
       serviceAccountName: gateway-api-controller
       terminationGracePeriodSeconds: 10
+      volumes:
+      - name: webhook-cert
+        secret:
+          defaultMode: 420
+          secretName: webhook-cert
```

Comment:
Why is all the webhook stuff only showing up now? Did we miss it before somehow?

Reply:
Discussed offline. This is expected since we baseline this diff off the v1.0.1.yaml. In future, we should consider iterating on the prev revision instead to reduce the size of this diff.